RELEASENOTES.md

```diff
@@ -41,6 +41,77 @@ repository but is still accessible under the
 
 ---
 
+### Networking Recipe
+
+The networking recipe enhances and extends the functionality enabled in 23.01
+release.
+
+#### Features and enhancements
+
+* Intel IPU E2100 target support
+* IPsec offload support (Intel IPU E2100 target only)
+* IPv6 support in infrap4d networking stack (Intel IPU E2100 target only)
+* Local mirroring (Intel IPU E2100 target only)
+* Fedora 37 support
+* ACC support for Rocky Linux 9.1 container
+* Cross-compile for ARM64 Rocky Linux 9.1
+* Multi-process TDI stack support (ability to run infrap4d as secondary process
+in multiple control planes environment)
+* Support for remote gRPC clients (infrap4d on ACC and gRPC clients on host)
+* Unit test framework
+* Security enhancements
+  * Introduction of `sgnmi_cli`, a secure-by-default
+gNMI client
+  * Library updates to address CVE security issues
+* Build script updates
+* Documentation updates
+* Bug fixes
+
+#### Limitations
+
+* The networking recipe currently uses a fork of `p4runtime` repository in order
+to support mirroring for Intel IPU E2100
+* Local mirroring feature: Modification for mirror packet does not work after
+first rule
+* IPv6 feature:
+  * Add a route with remote TEP IPv6 as nexthop
+
+    ```bash
+    ip -6 route change <local TEP network address> via <remote TEP address>
+    dev <idpf interface name corresponding to phy port 0>
+    
+    Example: ip -6 route change 1000:1::/64 via 1000:1::2 dev ens802f0d1
+    ```
+
+  * Only underlay IPv6 and overlay IPv6 use case are supported. IPv4-in-IPv6 and
+  IPv6-in-IPv4 support will be added in future release
+  * At any point either v4 or v6 tunnel is supported, due to above limitation.
+  * Configure vsi to vsi_group mapping by FXP SEM register due to missing
+  Control Plane Channel (CPCHNL) support Map the host1_vsi to vsi_group 1 by
+  executing following commands on IMC.
+  Note: VSI ID of host1 can be queried using
+  `./cli_client --query --verbose --config`
```

Comment:
There are multiple instances of cli_client on IMC right now. Latest is /usr/bin/cli_client. Please change the cmd to `/usr/bin/cli_client --query --verbose --config`